lang/java/pom.xml

```diff
@@ -289,9 +289,10 @@
               <!-- Avro uses Sun's java code style conventions with 2 spaces, this is a modified version of
                    the eclipse formatter -->
               <file>${main.basedir}/lang/java/eclipse-java-formatter.xml</file>
-              <version>4.19.0</version>
+              <version>4.21.0</version>
             </eclipse>
-            <removeUnusedImports/>
+            <!-- Temporarily disabled for JDK 16+ builds -->
+            <!--<removeUnusedImports/>-->
```

Comment:
I hope we can leave with this being disabled until we update Avro to JDK 11. Then we can add the needed `--add-exports ...` for Spotless Maven plugin in `.mvn/jvm.config`